.github/workflows/mvnw.yml

```diff
@@ -21,5 +21,13 @@ jobs:
           key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
           restore-keys: |
            ${{ runner.os }}-maven-
+      - name: Test token
```

Comment:
Shouldn't echo the token where it will be visible.

Reply:
tokens aren't visible in all CIs that we use, removed this anyways since it was for testing only